components/feedback_modal/templates/how_can_we_help.jade

```diff
@@ -4,6 +4,8 @@ h1.fm-garamind-headline
   | How may we help you?
 
 nav.small-chevron-list.fm-multiple-choice
+  a( data-answer='auction' )
+    | I have auctions-related question.
```

Comment:
AHHH my bad! "I have an auction-related question." thank you so much for spotting @sweir27 cc @craigspaeth